connectors/http/rest/README.md

```diff
@@ -87,8 +90,8 @@ The response will contain the status code, the headers and the body of the respo
   "url": "https://httpbin.org/basic-auth/user/password",
   "authentication": {
     "type": "basic",
-    "username": "{{secrets.USERNAME}}",
-    "password": "{{secrets.PASSWORD}}"
+    "username": "",
```

Comment:
all the `{{secrets.****}}` have been deleted despite being present in the .peb file. Other than that I think it's fine to be merged 👌